templates/service-template.yml

```diff
@@ -568,6 +568,11 @@ parameters:
   displayName: Kas-fleetshard operator subscription starting config
   description: Kas-fleetshard operator subscription starting config
 
+- name: MAX_ALLOWED_TRIAL_INSTANCES
+  displayName: Max allowed trial instances config 
+  description: Max allowed trial instances config
```

Comment:
The descriptions should be more specific, are we talking about max trial instances per kas fleet manager? per account? per organization?
Also I guess the concept of `trial` does not apply in kfm code, it should be eval instead? What do you think @machi1990 ?

Reply:
I agree and I believe that both `MAX_ALLOWED_TRIAL_INSTANCES` and `MaxAllowedTrialInstance` should be renamed to `MAX_ALLOWED_DEVELOPER_INSTANCES` and `MaxAllowedDeveloperInstance`

Reply:
+1. I think the flags should be specific to `developer` instances, `trial` instance is a different thingy which may come at a later stage.

Reply:
As I'm exploring KFM. Don't know much knowledge about KFM.
Can someone explain to me, would this flag will be per kfm or per  account or per organization  ?

Reply:
The flag is per user.

as a user, one can create up to `N` defined max instances if they do not have quota to create standard instances.